### PR TITLE
6016 enable cr in sqlite

### PR DESCRIPTION
**Pull request by gdicristofaro**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbManager.java [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+    }
+
+    /**
+     * attains central repository lock
+     *
+     * @param db the database connection
+     * @return the lock if acquired
+     * @throws CentralRepoException
+     */
+    private static CoordinationService.Lock attainCentralRepoLock(CentralRepository db) throws CentralRepoException {
```
[the remark this answers is not in the document]
Comment (gdicristofaro) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbManager.java]: oops.  will do.

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbManager.java [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+    private final SqliteCentralRepoSettings dbSettingsSqlite;
+
+    private boolean configurationChanged = false;
+
+    public CentralRepoDbManager() {
+        dbSettingsPostgres = new PostgresCentralRepoSettings();
+        dbSettingsSqlite = new SqliteCentralRepoSettings();
+        selectedPlatform = CentralRepoPlatforms.getSelectedPlatform();
+        if (selectedPlatform == null || selectedPlatform.equals(CentralRepoPlatforms.DISABLED)) {
+            selectedPlatform = CentralRepoPlatforms.POSTGRESQL;
```
[the remark this answers is not in the document]
Comment (gdicristofaro) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbManager.java]: This originally came from the EamDbSettingsDialog constructor before I divided up the code.  https://github.com/sleuthkit/autopsy/blob/b805d2a1ec3f4bc16229a900a8d3859ba3355758/Core/src/org/sleuthkit/autopsy/centralrepository/optionspanel/EamDbSettingsDialog.java  

I think it is so the drop down populates with something if no choice is selected.  Is this something that needs to change all together?

[the remark this answers is not in the document]
Comment (APriestman) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbManager.java]: Ok it's probably fine but please add a comment about what's going on.

[the remark this answers is not in the document]
Comment (gdicristofaro) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbManager.java]: Ok.  It should be in the PR now.

**Comment by APriestman** [comment]
Please remove the unused imports reported by Codacy.

**Pull request merged**